docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-modify-proc.adoc

```diff
@@ -0,0 +1,9 @@
+[id='environment-authoring-ha-modify-proc']
+= Modifying the template for the High Availability authoring environment
+
+By default, the High Availability authoring template creates a MySQL pod to provide the database backend. If you prefer to use PostgreSQL or to use an external server (outside the OpenShift project), you need to modify the template before deploying the environment.
```

Comment:
Is "High Availability" in the template name? If not, lowercase and hyphenated, as you say "high-availability authoring" and the like elsewhere. If the name of the template, then good.